src/main.cc

```diff
@@ -136,8 +136,10 @@ signal_sigwinch_cb(ev_loop *loop, struct ev_signal *w, int revents)
 	(void) loop;
 	(void) w;
 	(void) revents;
+#if defined(HAVE_GNU_READLINE)
```

Comment:
What a problem you fix here?

Reply:
Built-in mac readline does not contain `rl_resize_terminal`